src/main/java/com/pi4j/devices/is31Fl37Matrix/MonitorInterrupt.java

```diff
@@ -74,14 +75,18 @@ public static void main(String[] args) throws Exception {
                 i++;
             } else if (o.contentEquals("-d")) { // mainChip
                 pullDirection = args[i + 1];
-                if (pullDirection.contentEquals("UP") | pullDirection.contentEquals("DOWN") | pullDirection.contentEquals("OFF")) {
```

Comment:
in such situations you should use the double ||, as this short circuits the OR-ing.

Reply:
Maybe even think of using a switch here? Another idea would to go with the splendid https://picocli.info/ library